DuckDuckGo/Suggestion.swift

```diff
@@ -18,9 +18,22 @@
 //
 
 import Foundation
+import Core
 
-struct Suggestion {
+struct Suggestion: Decodable {
 
     let type: String
     let suggestion: String
+
+    init(from decoder: Decoder) throws {
```

Comment:
Needed a custom init function here since the `type` and `suggestion` values are derived from a single key/value pair in the dictionary. Is there a cleaner way to handle this?

Reply:
I think this is fine :)